src/main.ts

```diff
@@ -224,6 +225,29 @@ function isSupportedPlatform(platform: PlatformInformation): boolean {
 }
 
 async function ensureRuntimeDependencies(extension: vscode.Extension<CSharpExtensionExports>, eventStream: EventStream, platformInfo: PlatformInformation, installDependencies: IInstallDependencies): Promise<boolean> {
+    await activateSdkExtension(eventStream, platformInfo);
     return installRuntimeDependencies(extension.packageJSON, extension.extensionPath, installDependencies, eventStream, platformInfo);
 }
 
+async function activateSdkExtension(eventStream: EventStream, platformInfo: PlatformInformation) {
+    const sdkExtension = vscode.extensions.getExtension("ms-dotnettools.vscode-dotnet-sdk");
+    if (sdkExtension !== undefined) {
+        // Activate extension.
+        await sdkExtension.activate();
+
+        // Ensure SDK is installed.
+        eventStream.post(new PackageInstallation(".NET 5.0 SDK"));
+        const aquisitionResult = await vscode.commands.executeCommand<{ dotnetPath: string }>('dotnet-sdk.acquire', { version: '5.0' });
```

Comment:
I believe just calling acquire here will work, do we still need a new command from the SDK extension to check install status? The functional difference would be that the acquire command will cause a SDK to be installed if it hasn't been already, whereas we could implement a new command that will return 'undefined' or something like that if we aren't installing anything to signal that no SDK has been installed.